Libraries/WebSocket/RCTWebSocket.xcodeproj/project.pbxproj

```diff
@@ -433,6 +433,11 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				EXECUTABLE_PREFIX = lib;
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"DEBUG=1",
```

Comment:
Why did we add DEBUG here?

Reply:
@shergin When I added the GCC preprocessor section in XCode, it was added automatically, likely because this is the `debug` preset  (see line 440). My guess is that if you omit the preprocessor section XCode auto-assigns the DEBUG define, but I am no XCode/Objective-C wizard and can't confirm this.